guides/Getting Started.md

```diff
@@ -0,0 +1,696 @@
+The database for Friends.Repo has been created.
+```
+
+**NOTE:** If you get an error, you should try changing your configuration in `config/config.exs`, as it may be an authentication error.
+
+A database by itself isn't very queryable, so we will need to create a table within that database. To do that, we'll use what's referred to as a _migration_. If you've come from Active Record (or similar), you will have seen these before. A migration is a single step in the process of constructing your database.
+
+Let's create a migration now with this command:
+
+```
+mix ecto.gen.migration create_people
+```
+
+This command will generate a brand new migration file in `priv/repo/migrations`, which is empty by default:
+
+```elixir
+defmodule Friends.Repo.Migrations.CreatePeople do
+  use Ecto.Migration
+
+  def change do
+
+  end
+end
+```
+
+Let's add some code to this migration to create a new table called "people", with a few columns in it:
+
+```elixir
+defmodule Friends.Repo.Migrations.CreatePeople do
+  use Ecto.Migration
+
+  def change do
+    create table(:people) do
+      add :first_name, :string
+      add :last_name, :string
+      add :age, :integer
+    end
+  end
+end
+```
+
+This new code will tell Ecto to create a new table called people, and add three new fields: `first_name`, `last_name` and `age` to that table. The types of these fields are `string` and `integer`. (The different types that Ecto supports are covered in the [Ecto.Schema](https://hexdocs.pm/ecto/Ecto.Schema.html) documentation.)
+
+**NOTE: The naming convention for tables in Ecto databases is to use a pluralized name.**
+
+To run this migration and create the `people` table in our database, we will run this command:
+
+```
+mix ecto.migrate
+```
+
+If we found out that we made a mistake in this migration, we could run `mix ecto.rollback` to undo the changes in the migration. We could then fix the changes in the migration and run `mix ecto.migrate` again. If we ran `mix ecto.rollback` now, it would delete the table that we just created.
+
+We now have a table created in our database. The next step that we'll need to do is to create the schema.
+
+## Creating the schema
+
+The schema is an Elixir representation of data from our database. Each schema doesn't necessarily need to be tied to a table; they can be tied to database views too.
+
+Let's create the schema within our application at `lib/friends/person.ex`:
+
+```elixir
+defmodule Friends.Person do
+  use Ecto.Schema
+
+  schema "people" do
+    field :first_name, :string
+    field :last_name, :string
+    field :age, :integer
+  end
+end
+```
+
+This model defines the schema from the database that this model maps to. In this case, we're telling Ecto that the `Friends.Person` model maps to the `people` table in the database, and the `first_name`, `last_name` and `age` fields in that table. The second argument passed to `field` tells Ecto how we want the information from the database to be represented in our model.
```

Comment:
Let's replace "model" with "schema".

I think a paragraph about the relation of ecto types and database types could be a good idea as well.
Ecto types are used in schema and define how the data should be represented in elixir. Types in migrations correspond to database types - both does not need to match necessarily.